data/json/items/resources/misc.json

```diff
@@ -32,5 +48,35 @@
     "color": "white",
     "flags": [ "NONCONDUCTIVE" ],
     "use_action": { "type": "deploy_furn", "furn_type": "f_floor_mattress" }
+  },
+  {
+    "id": "chem_potassium_alum",
+    "type": "AMMO",
+    "category": "chems",
+    "name": "potassium alum",
+    "description": "Small white crystals of potassium alum with watery metallic odor, soluble in water, historically used for many different purposes.  Also known as potash alum and potassium aluminum sulfate; often called simply alum, as it's the most important member of the generic class of compounds called alums.",
+    "container": "bottle_plastic_small",
+    "weight": 1,
+    "volume": "250 ml",
+    "price": 2500,
+    "material": "powder_nonflam",
+    "symbol": "=",
+    "color": "dark_gray",
+    "count": 250,
+    "ammo_type": "components"
+  },
```

Comment:
I'd suggest making a new resource file, "chemicals.json", and putting this (and maybe dye powder) in there. We're going to need a separate file for chemicals eventually anyway.